keyboards/und/pcm-t-k0-windows.xml

```diff
@@ -95,6 +96,7 @@
 		<map iso="C09" to="L"/>
 		<map iso="C10" to="Ọ"/>
 		<map iso="C11" to="Ẹ"/>
+		<map iso="B00" to=""/>
```

Comment:
If this is an invisible character it needs to be escaped for clarity.

Reply:
Unless specially assigned, B00 should be the same as C12, e.g.,
		<map iso="C12" to="\"/> <!-- (key to right of ') -->
		<map iso="B00" to="\"/> <!-- (key to left of Z) -->

Reply:
Done